packages/openactive-broker-microservice/app.js

```diff
@@ -51,7 +51,7 @@ const WAIT_FOR_HARVEST = VALIDATE_ONLY ? false : config.get('broker.waitForHarve
 const VERBOSE = config.get('broker.verbose');
 const OUTPUT_PATH = config.get('broker.outputPath');
 
-const HARVEST_START_TIME = new Date();
+const HARVEST_START_TIME = (new Date()).toISOString();
```

Comment:
I changed this to a string rather than a Date because:

* More amenable to caching (which happens later on)
* It wasn't used anywhere as a Date. It was always converted into some other object first.
* I am suspicious of JS Dates and almost feel that they should never be used